prismic-model/src/articles.ts

```diff
@@ -7,80 +7,87 @@ import articleBody from './parts/article-body';
-  },
-  Contributors: contributorsWithTitle(),
-  Promo: {
-    promo,
-  },
-  Metadata: {
-    metadataDescription: structuredText('Metadata description', 'single'),
-  },
-  'Content relationships': {
-    series: list('Series', {
-      series: link('Series', 'document', ['series']),
-      positionInSeries: number('Position in series'),
-    }),
-    seasons: list('Seasons', {
-      season: link('Season', 'document', ['seasons'], 'Select a Season'),
-    }),
-    parents: list('Parents', {
-      order: number('Order'),
-      parent: link('Parent', 'document', ['exhibitions'], 'Select a parent'),
-    }),
-  },
-  Migration: {
-    publishDate: {
-      config: {
-        label: 'Override publish date',
-      },
-      type: 'Timestamp',
+const articles: CustomType = {
+  id: 'articles',
+  label: 'Story',
+  repeatable: true,
+  status: true,
+  json: {
+    Story: {
+      title,
+      format: link('Format', 'document', ['article-formats']),
+      body: articleBody,
     },
-    wordpressSlug: {
-      config: {
-        label: 'Wordpress slug',
-      },
-      type: 'Text',
+    Outro: {
+      outroResearchItem: link('Outro: Research item'),
+      outroResearchLinkText: singleLineText('Outro: Research link text'),
+      outroReadItem: link('Outro: Read item'),
+      outroReadLinkText: singleLineText('Outro: Read link text'),
+      outroVisitItem: link('Outro: Visit item'),
+      outroVisitLinkText: singleLineText('Outro: Visit link text'),
+    },
+    Contributors: contributorsWithTitle(),
+    Promo: {
+      promo,
     },
-    // TODO: deprecate
-    contributorsDeprecated: {
-      type: 'Slices',
-      fieldset: 'Contributors',
-      config: {
-        choices: {
-          person: {
-            type: 'Slice',
-            fieldset: 'Person',
-            'non-repeat': {
-              role: {
-                type: 'Link',
-                config: {
-                  label: 'Role',
-                  select: 'document',
-                  customtypes: ['editorial-contributor-roles'],
-                  tags: ['editorial'],
+    Metadata: {
+      metadataDescription: structuredText('Metadata description', 'single'),
+    },
+    'Content relationships': {
+      series: list('Series', {
+        series: link('Series', 'document', ['series']),
+        positionInSeries: number('Position in series'),
+      }),
+      seasons: list('Seasons', {
+        season: link('Season', 'document', ['seasons'], 'Select a Season'),
+      }),
+      parents: list('Parents', {
+        order: number('Order'),
+        parent: link('Parent', 'document', ['exhibitions'], 'Select a parent'),
+      }),
+    },
+    Migration: {
+      publishDate: {
+        config: {
+          label: 'Override publish date',
+        },
+        type: 'Timestamp',
+      },
+      wordpressSlug: {
+        config: {
+          label: 'Wordpress slug',
+        },
+        type: 'Text',
+      },
```

Comment:
To discuss separately: we don't read this value anywhere.

```console
$ grep --exclude-dir node_modules --exclude-dir .next --exclude-dir lib -rnI wordpressSlug .
./prismic-model/src/articles.ts:52:    wordpressSlug: {
./prismic-model/src/series.ts:44:    wordpressSlug: text('[Deprecated] Wordpress slug'),
./cardigan/stories/content.js:325:    wordpressSlug: null,
```

Can we delete it? If not, can we add a comment explaining why it's important? (e.g. it has historical significance)